src/styles/_services.scss

```diff
@@ -102,6 +108,17 @@
     }
     #below-45 {
       background: $finsec-kashmir-blue;
+      > div {
+        > div {
+          padding-bottom: 6vw !important;
```

Comment:
It's ok to us `!important` at the end of a hierarchy. Or when mobile, desktop and inline jQuery styles need to overwrite each other quite dramatically. I don't find it a drama, but many other devs do...